src/libs/actions/IOU.js

```diff
@@ -178,6 +178,8 @@ function requestMoney(report, amount, currency, recipientEmail, debtorEmail, com
                 lastVisitedTimestamp: Date.now(),
                 lastReadSequenceNumber: newSequenceNumber,
                 maxSequenceNumber: newSequenceNumber,
+                lastMessageText: optimisticReportAction.message.text,
```

Comment:
message is an array so we need to access:
```suggestion
                lastMessageText: optimisticReportAction.message[0].text,
```